std/traits.d

```diff
@@ -9143,9 +9143,10 @@ enum isType(alias X) = is(X);
 }
 
 /**
- * Detect whether symbol or type `X` is a function. This is different that finding
- * if a symbol is callable or satisfying `is(X == function)`, it finds
- * specifically if the symbol represents a normal function declaration, i.e.
+ * Detect whether symbol or type `X` is a function or function type.
+ * This is different from finding
+ * if a symbol is callable or satisfying `is(X == return)`. It finds
```

Comment:
No need for a newline there, but do put one at the end of the sentence.

Reply:
OK fixed, thanks. It seems GitHub has gotten smarter about sub-line diffs, which is great! That's why I had formatted it like that, but it seems that's no longer needed.